TradeItIosTicketSDK2/TradeItPreviewTradeOrderDetails+Helpers.swift

```diff
@@ -1,31 +1,122 @@
-@objc public enum TradeItOrderExpiration: Int {
+public typealias RawValue = String
+
+@objc public enum TradeItOrderExpiration: Int, RawRepresentable {
     case goodForDay
     case goodUntilCanceled
     case unknown
+    
+    public var rawValue: RawValue {
+        switch self {
+        case .goodForDay:
+            return "day"
+        case .goodUntilCanceled:
+            return "gtc"
+        default:
+            return "unknown"
+        }
+    }
+    
+    public init?(rawValue: RawValue) {
+        self.init(value: rawValue)
+    }
+    
+    public init(value: RawValue?) {
```

Comment:
Do these work for Obj-C users?

Reply:
No, in Objective C, you can use only the enum `TradeItOrderActionBuy`, ...